src/components/signup/SignupState.js

```diff
@@ -317,17 +307,10 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
           // set profile data
           await userStorage.setProfile(profile)
 
-          log.debug('adding tokens', { tokens: fromPairs(tokenFields) })
-
-          // set tokens
-          await Promise.all(
-            tokenFields.map(([fieldName, fieldValue]) => userStorage.setProfileField(fieldName, fieldValue, 'private')),
-          )
-
           log.debug('setting registered flag')
 
           // set registered flag
-          await Promise.all([userProperties.set('registered', true), AsyncStorage.removeItem(GD_INITIAL_REG_METHOD)])
+          userProperties.set('registered', true)
```

Comment:
calling .set() is prohibited due to the possible uncaught promise rejection
we have to await set or to calll safeSet() having .catch with log.warn inside

Reply:
will use await here as we're awaiting for persist anyway